docs/guide/backend-integration.md

```diff
@@ -1,6 +1,8 @@
 # Backend Integration
 
-If you want to serve the HTML using a traditional backend (e.g. Rails, Laravel) but use Vite for serving assets, here's what you can do:
+If you want to serve the HTML using a traditional backend (e.g. Rails, Laravel) but use Vite for serving assets, check for existing integrations listed in [Awesome Vite](https://github.com/vitejs/awesome-vite#integrations-with-backends).
+
+Else, you can follow these steps to configure it manually:
```

Comment:
```suggestion
Or you can follow these steps to configure it manually:
```

Not a native speaker, but `Else,` looks a bit uncommon to me. WDYT?

Reply:
It's not uncommon, but either way is fine.